cli/cli.py

```diff
@@ -0,0 +1,181 @@
+        return
+    from cameo import __version__
+    click.echo(__version__)
+    ctx.exit()
+
+
+@click.group()
+@click.option('--version', is_eager=True, callback=print_version, default=False, is_flag=True,
+              help='Display version information.')
+def main(version):
+    """cameo command line interface"""
+    pass
+
+
+@main.command()
+@click.option('-o', '--output', default='-', multiple=True, type=click.Path(),
+              help='Output filename. Multiple output files can be provided (pair with respective format options).')
+@click.option('-f', '--format', multiple=True, default=['csv'],
+              type=click.Choice(VALID_OUTPUT_FORMATS), help='Output file format (default csv).')
+@click.option('-h', '--host', default=['ecoli'], multiple=True,
+              type=click.Choice(['ecoli', 'scerevisiae']),
+              help='The host organisms to consider (default: all). '
+                   'Multiple hosts can be specified by repeating --host HOST')
+@click.option('--aerobic', default=True, is_flag=True, help='Make oxygen available to the host organism (default).')
+@click.option('--cores', default=1, type=click.IntRange(1, multiprocessing.cpu_count()),
+              help='Number of CPU cores to use (default 1).')
+@click.option('--differential-fva/--no-differential-fva', default=True,
+              help='Perform differential flux variability analysis to determine flux modulation targets (default).')
+@click.option('--heuristic-optimization/--no-heuristic-optimization', default=True,
+              help='Find gene knockout targets through heuristic optimization (default).')
+@click.option('--max-pathway-predictions', default=1,
+              help='Maximum number of heterologous pathways to predict (default 1).')
+@click.option('--differential-fva-points', default=10, help='Grid points for differential FVA (default 10).')
+@click.option('--pathway-prediction-timeout', default=10 * 60,
+              help='Time limit (min) for individual pathway predictions (default 10 min).')
+@click.option('--heuristic-optimization-timeout', default=45,
+              help='Time limit (min) on individual heuristic optimizations (default 45 min).')
+@click.option('--logging', default='ERROR',
+              type=click.Choice(['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG']),
+              help='Logging level (default WARNING).')
+@click.argument('product')
+def design(product, host, output, format, cores, aerobic, differential_fva, heuristic_optimization,
+           max_pathway_predictions, differential_fva_points, pathway_prediction_timeout,
+           heuristic_optimization_timeout, logging):
+    """Compute strain designs for desired product."""
+    from cameo.api.designer import design
+    from cameo.api.hosts import hosts
+    from cameo.parallel import MultiprocessingView, SequentialView
+
+    logger.setLevel(logging)
+    cameo_logger.setLevel(logging)
+    inspyred_logger.setLevel(logging)
```

Comment:
We should not change inspyred logger. Debugging a heuristic optimization library is for advanced users, not for the high-level users.